arm-resources/resources/2016-07-01/swagger/resources.json

```diff
@@ -1684,7 +1689,8 @@
           },
           "description": "The api versions."
         }
-      }
+      },
+      "description": "The type of alias paths. "
```

Comment:
is it alias paths (plural) or just path without the "s".  

Would it be clearer with something like:
     "The type of the alias for the path"